### PR TITLE
Refactor readme

### PR DESCRIPTION
**Pull request by jasonjkeller**
Holding for release. This can go out after the 7.0.0 Java agent release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,68 +45,192 @@ $ ./gradlew test
...
+
+The `jfr-daemon` standalone process has the following requirements:
+* An [Insights insert API key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#event-insert-key) or
+  an [APM license key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/new-relic-api-keys#ingest-license-key).
+  To use an APM license key you will need to configure the JFR daemon with `USE_LICENSE_KEY=true`.
+* Uses a [supported Java version](#supported-java-versions)
+
+#### Usage for standalone process
+
+The minimum requirements to use the `jfr-daemon` as standalone Java agent are as follows.
```
Comment (XiXiaPdx) [inline README.md:163]: I think meant `standalone process` here?

Comment (jasonjkeller) [reply README.md:163]: Yes! Good eye!

File: README.md [hunk 2]
```diff
@@ -48,68 +45,192 @@ $ ./gradlew test
...
+[system properties](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#System_Properties),
+and [environment variables](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#Environment_Variables).
+
+The following agent configuration options affect how and where JFR data is reported:
+* [app_name](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#cfg-app_name)
+* [license_key](https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file/#cfg-license_key)
+* [jfr.enabled]()
+* [jfr.audit_logging]()
+* [metric_ingest_uri]()
+* [event_ingest_uri]()
```
Comment (jasonjkeller) [inline README.md:139]: FYI these links are published yet. I'm planning on updating them when a batch of docs goes live.

**Review by XiXiaPdx — APPROVED** [review]
💯

**Pull request merged**